frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -112,7 +195,7 @@ const defaultProperties = ({
     const properties = filterGroup.values as AnyPropertyFilter[]
 
     return {
-        event: '$exception',
+        event: '$pageview',
```

Comment:
copy pasta?

Reply:
we should have expection autocapture turned on and a secret path that throws errors